### DISCUSSION TITLE
Running in a browser

### DISCUSSION THREAD

**Discussion by chensce** [vmware-labs/wasm-workers-server#206] [Ideas]
### Is your feature request related to a problem? Please describe.

If we can run the wasm workers server in browser, we can run it anywhere.

### Describe the solution you'd like

we know the browser (chrome)have Wasm runtime

**Comment by ereslibre** [comment]
👋 Thank you for your report!

It would be really great to run Wasm Workers Server in the browser! Wasm Workers Server at this time is a developer-focused tool that will look for Wasm Modules and supported languages extensions in the filesystem, and will serve these functions in HTTP endpoints created on the fly.

There are many ways in which we could map Wasm Workers Server to the browser, but we would like to understand your use case a bit better.

How do you expect to use Wasm Workers Server on the browser? Can you share some use cases with us, and how would you expect them to work?

Thank you :)

**Comment by chensce** [comment]
[webcontain](http://webcontainers.io)

**Comment by chensce** [comment]
[shell.duckdb.org](https://shell.duckdb.org/)

**Comment by Angelmmiguel** [comment]
EDIT: I found I can convert an issue to discussion directly 🤦. Sorry for the noise!

------------

~~Hello @gedw99, @ereslibre and @chensce!~~

~~Thank you for sharing all the links and ideas about running `wws` in the browser! This is an interesting idea and would love to continue discussing about it.~~

~~We recently enabled the "Discussions" section in the repository. I created a new "Idea" to continue the conversation there. I'll move the comments there, so don't worry about it!~~

~~https://github.com/vmware-labs/wasm-workers-server/discussions/205~~
